analysis/time_distribution_analysis/time_analysis_main.py

```diff
@@ -3,41 +3,32 @@
 import math
 import pprint
 
-from data_preparation.data_grouping import load_grouped_filtered_sstubs
 from constants import MIN_SSTUB_PERCENTAGE
+from utils import get_average
 
 
-def find_min_time_interval(timestamps_list, min_sstubs_percentage):
```

Comment:
the Diff looks more confusing than it is in this file unfortunately. I didn't change the existing logic really, just added a function at the top and moved `find_min_time_interval()` down.